neo/Network/P2P/Helper.cs

```diff
@@ -1,43 +1,40 @@
-﻿using K4os.Compression.LZ4.Streams;
+﻿using K4os.Compression.LZ4;
 using Neo.Network.P2P.Payloads;
 using System;
+using System.Buffers;
 using System.IO;
 
 namespace Neo.Network.P2P
 {
     public static class Helper
     {
-        public static byte[] DecompressLz4(this byte[] data, int maxOutput = int.MaxValue)
+        public static byte[] DecompressLz4(this byte[] data, int maxOutput)
         {
-            using (var input = new MemoryStream(data, false))
-            using (var decoder = LZ4Stream.Decode(input, leaveOpen: true))
-            using (var output = new MemoryStream())
+            maxOutput = Math.Min(maxOutput, data.Length * 255);
```

Comment:
This could be a lot of, i think that this was the reason for skip this mode, what do you think about add another parameter for `buffer`, in tcp we don't have paralell deserialization, so maybe we can reuse a static buffer

Reply:
static = variable inside RemoteNode

Reply:
They are re-using buffers from `ArrayPool<byte>`.

Reply:
I am concerned about if 5 nodes send a 1mb compressed message, we will allocate (1 * 255 * 5 )mb

Reply:
How does `ArrayPool<byte>.Shared` deal with this?

Reply:
Why 255 * 5 mb？We have `Math.Min()`.

Reply:
You are right, i was thinking that is a Max xD